clickhouse_std.go

```diff
@@ -82,9 +84,14 @@ func (o *stdConnOpener) Connect(ctx context.Context) (_ driver.Conn, err error)
 			}, nil
 		}
 	}
+
 	return nil, err
 }
 
+func (o *stdConnOpener) setClientInfoDefaults() {
+	o.opt.ClientInfo.comment = []string{"database/sql"}
```

Comment:
`database/sql` or `api:database/sql`? Maybe it would be easier to parse with `key:value` structure?

Reply:
I treat it more like a flag, that tells there is an additional abstraction layer used. We will have parse capability anyway.